### PR TITLE
add validation check for the input contact input form

### PR DESCRIPTION
**Pull request by Amutha37**
Did clean up  and please review this changes for contact input validation check. The input field for name will only allow [,-']. The email need to follow standard email address format.

### CODE REVIEW

File: src/components/contactForm/ContactForm.js [hunk 1]
```diff
@@ -1,32 +1,60 @@
...
+      <input
+        className="checkname"
+        type="text"
+        placeholder="name"
+        value={name}
+        pattern="[a-zA-Z]+(?:(?:\. |[' ])[a-zA-Z]+)*"
+        onChange={({ target }) => {
+          setName(target.value);
+        }}
+        autocomplete="on"
```
Comment (lucitemple) [inline src/components/contactForm/ContactForm.js]: Error in console - should this be autoComplete?

File: src/components/contactForm/ContactForm.js [hunk 2]
```diff
@@ -1,32 +1,60 @@
...
+      <input
+        className="checkemail"
+        type="text"
+        value={email}
+        // pattern=""
+        onChange={({ target }) => {
+          setEmail(target.value);
+        }}
+        placeholder="example@domain.com"
+        autocapitalize="off"
```
Comment (lucitemple) [inline src/components/contactForm/ContactForm.js]: Error in console - should this be autoCapitalize?

File: src/components/contactForm/ContactForm.js [hunk 3]
```diff
@@ -1,32 +1,60 @@
...
+        spellcheck="false"
```
Comment (lucitemple) [inline src/components/contactForm/ContactForm.js]: Error in console - should this be spellCheck?

File: src/components/contactForm/ContactForm.js [hunk 4]
```diff
@@ -1,32 +1,60 @@
...
+        inputmode="email"
```
Comment (lucitemple) [inline src/components/contactForm/ContactForm.js]: Error in console - should this be inputMode?

**Review by lucitemple — CHANGES_REQUESTED** [review]
Hi Amutha, thanks for sorting the validation. It's looking pretty good, however four warnings are coming up in the console:

Screen Shot 2021-05-08 at 7 55 19 pm
Screen Shot 2021-05-08 at 7 55 36 pm

Can you please address these and resubmit? Thanks.

**Comment by Amutha37** [comment]
If this is creating a contact list than no need autocomplete. If it is
editing than is ok to have auto complete. We are creating contacts so no
need autocomplete.

**Comment by lucitemple** [comment]
> If this is creating a contact list than no need autocomplete. If it is editing than is ok to have auto complete. We are creating contacts so no need autocomplete.

Hi Amutha, my comments are in reference to the errors in the console that occur when running the branch you've requested a pull request for. I'm not questioning whether we do or don't use autocomplete, but rather that the error is suggesting it might need to be `autoComplete`  rather than `autocomplete` (correct syntax to stop the error).

**Comment by Amutha37** [comment]
Sorry, I’m not at the computer now. I will have a look at it tomorrow or
Monday.
I just read your message clearly. It should be lowercase ‘c’.

Let me correct that when I get back to VSCode.
Thanks

File: src/components/contactForm/ContactForm.js (hunk 1 above)
Comment (Amutha37) [reply src/components/contactForm/ContactForm.js]: Yes, It should e autoComplete.

File: src/components/contactForm/ContactForm.js (hunk 2 above)
Comment (Amutha37) [reply src/components/contactForm/ContactForm.js]: Yes, It should be autoCapitalize.

File: src/components/contactForm/ContactForm.js (hunk 4 above)
Comment (Amutha37) [reply src/components/contactForm/ContactForm.js]: fixed

File: src/components/contactForm/ContactForm.js (hunk 3 above)
Comment (Amutha37) [reply src/components/contactForm/ContactForm.js]: fix to camel case

**Pull request closed without merging**